apps/leo_manager/priv/leo_manager_0.conf

```diff
@@ -86,9 +86,25 @@ consistency.rack_aware_replicas = 0
 ## A number of replication targets
 ## mdc_replication.max_targets = 2
 
-## A number of replicas a DC
+## A number of replicas per a datacenter
+## [note] A local LeoFS sends a stacked object which contains an items of a replication method:
+##          - [L1_N] A number of replicas
+##          - [L1_W] A number of replicas needed for a successful WRITE operation
+##          - [L1_R] A number of replicas needed for a successful READ operation
+##          - [L1_D] A number of replicas needed for a successful DELETE operation
+##       A remote cluster of a LeoFS system which receives its object,
+##       and then replicates it by its contained reoplication method.
```

Comment:
@mocchira In offline, as I mentioned,  ` N_Value >= MDCR_N_Value` is validated by another function which is `cluster-join`. I'm going to file its issue on LeoFS' repository and contain its comment on the configuration.